WHO.js

```diff
@@ -0,0 +1,192 @@
+{
+	"translatorID": "cd587058-6125-4b33-a876-8c6aae48b5e8",
+	"label": "WHO",
+	"creator": "Mario Trojan",
+	"target": "^http://apps\\.who\\.int/iris/",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2018-03-16 12:14:19"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2018 Mario Trojan
+	
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+
+// attr()/text() v2
+function attr(docOrElem,selector,attr,index){var elem=index?docOrElem.querySelectorAll(selector).item(index):docOrElem.querySelector(selector);return elem?elem.getAttribute(attr):null;}
+function text(docOrElem,selector,index){var elem=index?docOrElem.querySelectorAll(selector).item(index):docOrElem.querySelector(selector);return elem?elem.textContent:null;}
+function node(docOrElem,selector,index){var elem=index?docOrElem.querySelectorAll(selector).item(index):docOrElem.querySelector(selector);return elem?elem:null;}
+
+function detectWeb(doc, url) {
+	if (url.includes('/simple-search')) {
+		return "multiple";
+	}
+	
+	if (text(doc, "h2 > small") == "Collection home page") {
+		return "multiple";
+	}
+	
+	return "document";
+}
+
+
+function getSearchResults(doc, checkOnly) {
+	var items = {};
+	var found = false;
+	
+	var rows = doc.querySelectorAll('a.list-group-item');
+	for (let i=0; i<rows.length; i++) {
+		let href = rows[i].href;
+		Z.debug(href);
+		var title = text(rows[i], 'div.row > div.col-md-10 > h5');
```

Comment:
This looks fragile to me -- no attributes we could use to make it more robust?

Reply:
@mtrojan-ub Did you try
```js
var title = rows[i].textContent;
```

I expect this gives the title together with the year (No?), but that is okay for the list to choose from.